staff.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Staff
+---
+
+## Identifying Staff
+
+The _staff_ are the volunteers who act as network and system administrators to
+keep Libera.Chat running smoothly.
+
+The complete current list of staff is available using the IRC command
+`/who libera/staff/*`.
+
+When this page was last updated, the staff members were:
```

Comment:
I'm not sure this section is necessary. It seems prone to being out of date and inaccurate, as does (less so) the mention of bots below.

Reply:
I really, really do not want to have yet another place to keep in sync with a list of staff, and I really really don't think a table is the right choice.

This information is provided on `/about` which should be the only source

Reply:
I'm happy to drop the table; that's why I've kept it in separate commit.

And it might be useful elsewhere.